docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -33,6 +33,14 @@ The main features of the Media Controller API include:
 
   You can [set age rating for currently playing media](#setting_age_rating) by using updatePlaybackAgeRating() server method.
 
+- Managing playlists on server side
+
+  You can [managing playlists](#managing-playlists-on-server-side) on server side by creating, saving, deleting and get information about playlists. You can also update playback item from playlists item lists.
+
+- Managing playlists on client side
+
+  You can [managing playlists](#managing-playlists-on-client-side) on client side sending request about new playback item to server and adding listeners to be invoked when playlist is updated on server.
```

Comment:
Suggest rephrasing to:
You can [manage playlists](#managing-playlists-on-client-side) on client side by sending request about the new playback item to the server. You can then add listeners to be invoked when the playlist is updated on the server.

Reply:
Done